tests/modelbench_tests/test_benchmark_runner.py

```diff
@@ -146,6 +159,30 @@ def sut_response(self):
     def hazard(self):
         pass
 
+    def test_test_run_loads_annotators(self, tmp_path, item_from_test, benchmark):
```

Comment:
Ah, good stuff. Since the runner and the run are so tightly bound, I like the choice of testing it like this.